src/renderer/components/Substitution.vue

```diff
@@ -1,14 +1,30 @@
 <template>
-  <div class="crypto-tool-list">
+  <div class="view">
+    <div class="crypto-tool-list">
+    </div>
+    <div class="inputs">
+      <cipher-inputs-sub/>
+    </div>
   </div>
 </template>
 
 <script>
+  import cipherInputsSub from '@/components/cipher-inputs-sub'
   export default {
-    name: 'substitution'
+    name: 'substitution',
+    components: {
+      cipherInputsSub
+    }
   }
 </script>
+<style lang="scss">
+  .view {
+    display: flex;
```

Comment:
There's probably and standard way to share these kinds of styles between components in Vue. Not sure if external SCSS files is the way to go or what... Not necessarily asking for it to be changed (cuz I've done the same thing)... mostly making a note : )

Reply:
Yeah, I'm sure there MUST be a way to do that...